### ISSUE TITLE
Documentation Generation as Azure Pipeline Step

### ISSUE THREAD

**Issue by ChrisBlankDe** [365businessdev/vscode-alxmldocumentation#74]
Generating the docs on a devs notebook is cool but imho not the best solution from the point of view of process automation.
What do you think about providing an additional extension for azure pipelines that can provide the documentation as part of an automated build process?

im not a ts dev so maybe this is just bs:
Maybe move the common logics to an npm package which is comsumed by vscode and pipeline extension.

**Comment by 365businessdev** [comment]
Hello Chris,

Yes, you are absolutely right! We've been poking around for a while to find the best way to develop a pipeline task to run the AL XML documentation export and diagnostics. We've been looking for the best way to run the VS code extension and the DevOps pipeline task with the same core, but right now we haven't found the perfect way.

It's not about TypeScript or NPM, but we realized that we need to restructure the components to handle VS Code-specific things and the pipeline task. So we decided to look for the best technology and we are currently testing .net core for the core components.

Long story short, we plan to release both the VS Code extension and the DevOps extension later this year. So we are working on it, but you will have to be patient for a little while.
